cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -462,6 +462,17 @@ body classes ENT_5279
         kept_returncodes => { "0", "1" };
 }
 
+bundle agent cfengine_internal_cfengine_db
+{
+  users:
+      "cfpostgres"
+        policy => "present",
+        description => "CFEngine PostgreSQL",
+        group_primary => "cfpostgres",
+        home_dir => "/home/cfpostgres",
```

Comment:
Where should we place this? `/var/cfengine/pg/home`, `/opt/cfengine/home/cfpostgres` ?

Reply:
Nowhere, I don't think the *cfpostgres* user needs a home directory. The passing *FR-tests* in https://ci.cfengine.com/job/pr-pipeline/7716/ contain things like:
```
00:04:59 + ssh -o LogLevel=ERROR -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o ConnectTimeout=60 root@54.229.114.26 'su - cfpostgres --command "PGDATA=/var/cfengine/state/pg/data /var/cfengine/bin/pg_ctl status"'
00:05:00 su: warning: cannot change directory to /home/cfpostgres: No such file or directory
00:05:00 could not change directory to "/root": Permission denied
```
So it's noise, but harmless (in our cases). The proper thing to do is to `cd /tmp` first in all those commands which removes the second warning, the first one will go away when there's no home directory, AFAICT.

Reply:
NOTE TO SELF: This means, for all commands promises that use `psql` we need to add/extend a `contain` body that uses `chdir => "/tmp".